src/lib/containers/widgets/query-filter/RangeFacetFilter.tsx

```diff
@@ -19,6 +19,7 @@ export type RangeFacetFilterProps = {
   columnModel: ColumnModel
   onChange: Function
   facetAliases: {} | undefined
+  collapsed: boolean
```

Comment:
just wondering if it's intentional that it's optional in one EnumFilter and not here

Reply:
Not intentional--making both optional for consistency